ecl-patch/0001-Enable-utilisation-of-commercial-licenses.patch

```diff
@@ -87,8 +83,9 @@ index d8c3b17..232a1f2 100644
 +          oldStrategy(x)
 +      }
 +    )
-   }
- }
+       .enablePlugins(DockerPlugin, JavaAppPackaging)
+       .settings(
+       mappings in Universal += file("librpi_ws281x.so") -> "lib/librpi_ws281x.so",
```

Comment:
So this is the main change?

Reply:
Indeed. Thinking about your comment a bit... Maybe we should add some description of the process in the `ecl` script about the intention of this thing...